scout/server/blueprints/panels/templates/panels/panels.html

```diff
@@ -269,6 +272,11 @@ <h5 class="modal-title">Remove gene panel "{{panel.display_name}}"?</h5>
 <script src="https://cdn.datatables.net/1.12.0/js/jquery.dataTables.min.js" integrity="sha512-fu0WiDG5xqtX2iWk7cp17Q9so54SC+5lk/z/glzwlKFdEOwGG6piUseP2Sik9hlvlmyOJ0lKXRSuv1ltdVk9Jg==" referrerpolicy="no-referrer" crossorigin="anonymous"></script>
 <script type="text/javascript">
 
+var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
+var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
```

Comment:
Do you need to retain these variables? Or would it work to:

```
document.querySelectorAll('[data-bs-toggle="tooltip"]').forEach(function (tooltipTriggerEl) {
  new bootstrap.Tooltip(tooltipTriggerEl);
});
```

If the aim is to just initiate the tooltips. Just curious!

Reply:
I think I might have simply copied it from another page. I'll test what you suggest

Reply:
Your were right, it's not needed. I removed it now. Thanks!